lib/query-interface.js

```diff
@@ -883,6 +883,25 @@ module.exports = (function() {
     return this.sequelize.query(sql, options);
   };
 
+  QueryInterface.prototype.deferConstraints = function (transaction, options) {
+    if (!transaction || !(transaction instanceof Transaction)) {
+      throw new Error('Unable to start a transaction without transaction object!');
+    }
+
+    options = Utils._.extend({
+      transaction: transaction,
+      parent: options.transaction
+    }, options || {});
+
+    var sql = this.QueryGenerator.deferConstraintsQuery(options);
+
+    if (sql) {
+      return this.sequelize.query(sql, options);
+    } else {
```

Comment:
As I prefer the same result type of a function no matter what I would rather not remove the else. This way it always returns a `promise` and not `null or a promise`